tests/Plivo.Test/Resources/TestMessage.cs

```diff
@@ -29,21 +28,39 @@ public void TestMessageCreate()
 
             var response =
                 System.IO.File.ReadAllText(
-                    SOURCE_DIR + @"Mocks/messageSendResponse.json"
+                    SOURCE_DIR + @"../Mocks/messageSendResponse.json"
                 );
             Setup<MessageCreateResponse>(
                 201,
                 response
             );
-            Assert.IsEmpty(
+            Assert.Empty(
                 ComparisonUtilities.Compare(
                     response,
-                    Api.Message.Create( new List<string>() {"+919898989898", "+919090909090"},
-                        "textext", "+919999999999")));
+                    Api.Message.Create(src:"+919999999999", dst:new List<string>() {"+919898989898", "+919090909090"},
+                       text:"textext")));
+
             AssertRequest(request);
+            Assert.Empty(
+               ComparisonUtilities.Compare(
+                   response,
+                   Api.Message.Create(src: "+919999999999", dst:  "+919898989898<+919090909090",
+                      text: "textext")));
+
+            Assert.Empty(
+               ComparisonUtilities.Compare(
+                   response,
+                   Api.Message.Create(dst: "+919898989898<+919090909090",
+                      text: "textext",powerpack_uuid:"asdasd-asdasd-asdasd21-asd")));
+
+            Assert.Empty(
+               ComparisonUtilities.Compare(
+                   response,
+                   Api.Message.Create(dst: new List<string>() { "+919898989898", "+919090909090" },
```

Comment:
same here

Reply:
Yes, that's the idea to support both String and List.